webapp-client/src/errorify.js

```diff
@@ -0,0 +1,30 @@
+import ErrorMessage from './component/ErrorMessage';
```

Comment:
Please include this in the PR description. For example: Render items and add global error function.

Reply:
It has its own PR in #46, so I guess this actually should be based on that.

Reply:
Got it. Do you think you can merge #46 first then before this one?

Reply:
Yes!